public/js/actions/sources.js

```diff
@@ -51,6 +52,19 @@ function newSource(source) {
   };
 }
 
+function loadSourceMapSources(tab, source, dispatch) {
+  loadSourceMap(tab, source).then(map => {
+    const originalUrls = getOriginalSources(map);
+    let inc = 1;
```

Comment:
A UUID might be better so it doesn't depend on any state (not a blocker for now)